### ISSUE TITLE
Events

### ISSUE THREAD

**Issue by ChrisSchneider** [Integreat/ios-prototype#6]

**Comment by svenseeberg** [comment]
I suppose that this issue is about the not working events? I was just contacted by Bad Tölz and they reported this problem.

**Comment by ChrisSchneider** [comment]
Yes, it's something that is not implemented yet - but should be available with the Xamarin app
